Framework/Source Files/Model/Peripheral/ConnectablePeripheral.swift

```diff
@@ -14,10 +14,11 @@ public extension Peripheral where Type == Connectable {
     }
     
     /// Deafult initializer for Perpipheral.
-    /// - Parameter configuration: proviously created configuration containing all desired services and characteristics.
-    /// - Parameter deviceIdentifier: optional parameter. If device identifier is cached locally than it should be passed here.
-    ///   When set, connection to peripheral is much quicker.
-    /// - SeeAlso: Configuration
+    ///
+    /// - Parameters:
+    ///     - configuration: proviously created configuration containing all desired services and characteristics.
```

Comment:
Typo in `proviously`
```suggestion
    ///     - configuration: previously created configuration containing all desired services and characteristics.
```